src/QuadForm/Herm/GenusRep.jl

```diff
@@ -48,86 +8,171 @@ end
 
 ################################################################################
 #
-#  Genus representatives for hermitian lattices
+#  Neighbours methods
 #
 ################################################################################
 
+function _all_row_span(M)
```

Comment:
It appears this function is not called. Maybe we can add an example where this is used?

But I am also happy to improve the coverage later if you want to first get this in.

Reply:
For now yes, it is used in a specific loop of `_neighbours` that is not covered because it needs some specific examples I guess... Then we can wait before merging so that I can try to find/construct a nice example that will target this specific loop.